starepandas/tools/trixel_conversions.py

```diff
@@ -6,7 +6,7 @@
 import pystare
 import shapely
 import geopandas
-
+from shapely.geometry import Point
```

Comment:
seems common practice, but I am personally not a huge fan!

Reply:
feel free to modify

Reply:
I can do this

Reply:
nvm

Reply:
hahaha. Yeah; I was more testing out githubs reviewing tool than anything else.